src/NodaTime/XmlSchemaDefinition.cs

```diff
@@ -0,0 +1,241 @@
+// Copyright 2020 The Noda Time Authors. All rights reserved.
+// Use of this source code is governed by the Apache License 2.0,
+// as found in the LICENSE.txt file.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Reflection;
+using System.Xml;
+using System.Xml.Schema;
+using System.Xml.Serialization;
+using NodaTime.Extensions;
+
+namespace NodaTime
+{
+    /// <summary>
+    /// Provides XML schema types.
+    /// </summary>
+    public static class XmlSchemaDefinition
+    {
+        /// <summary>
+        /// Gets the XML namespace for all NodaTime types.
+        /// </summary>
+        /// <remarks>See [Namespaces in XML 1.1 (Second Edition)](https://www.w3.org/TR/xml-names11/).</remarks>
+        public static readonly XmlQualifiedName NodaTimeXmlNamespace = new XmlQualifiedName("nodatime", "https://nodatime.org/api/");
```

Comment:
Please make this a property - at 2.0 I removed all public static fields, even readonly ones.

```suggestion
        public static XmlQualifiedName NodaTimeXmlNamespace { get; } = new XmlQualifiedName("nodatime", "https://nodatime.org/api/");
```

(Unless there's some particular reason it needs to be a field.)

Reply:
There was no particular reason, I changed it to a property.